CMakeLists.txt: Fix Python install path

It seems Python already prepends MacPorts' install prefix by default, so
there's no need to add ${CMAKE_INSTALL_PREFIX} a second time.

Upstream-Status: Inappropriate [configuration]
--- ./CMakeLists.txt.orig	2023-11-15 13:44:07
+++ ./CMakeLists.txt	2023-11-15 13:44:39
@@ -409,7 +409,7 @@
       ${CMAKE_COMMAND} -E env
         CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}
         ${Python3_EXECUTABLE} ${SETUP_PY} install
-        --root $ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}
+        --root $ENV{DESTDIR}
       RESULT_VARIABLE STATUS)
     if (NOT STATUS STREQUAL 0)
       message(FATAL_ERROR \"pywatchman install failed\")
